locale/en_US/common.po

```diff
@@ -1757,6 +1757,27 @@ msgstr "Navigation menu was succesfully updated"
 msgid "notification.removedNavigationMenu"
 msgstr "Navigation menu was succesfully removed"
 
+msgid "notification.unsubscribeNotifications.success"
+msgstr "You have been unsubscribed"
+
+msgid "notification.unsubscribeNotifications.error"
+msgstr "We could not unsubscribe you"
+
+msgid "notification.unsubscribeNotifications.pageMessage"
+msgstr "Select the emails that you no longer wish to receive at {$email} from {$contextName}."
+
+msgid "notification.unsubscribeNotifications.resubscribe"
+msgstr "You can resubscribe to email notifications at any time from your <a href='{$profileNotificationUrl}'>user profile</a>."
```

Comment:
Use `"` for HTML attribute value quoting. (Also below)

Reply:
Fixed